### PR TITLE
Seed file to create cities, trip_estimates and users

### PR DESCRIPTION
**Pull request by akilchomoko**
Creates 1000 (out of 6400) city_airports
Creates 5000 trip estimates against those 1000 city_airports - valid from 1/4/2021 to 1/4/2022 for random prices/flight_times
Creates 8 users with password as set in the seed file

### CODE REVIEW

**Comment by akilchomoko** [comment]
Note: I've noticed an error in the seed file (delete order) plus too many trip estimates based on the number of users. However I can't send another commit due to following error:

To github.com:gtm19/up-in-the-air.git
 ! [rejected]        seed -> seed (fetch first)
error: failed to push some refs to 'git@github.com:gtm19/up-in-the-air.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

Tried Git pull but got error on schema. Need to talk through.

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,138 @@
...
+    city = City.all.sample
+    u = User.new
+    u.first_name = user[:firstname]
+    u.last_name = user[:lastname]
+    u.email = "#{user[:firstname]}.#{user[:lastname]}@upintheair.com"
+    u.password = "topsecret"
+    u.address_1 = user[:address1]
+    u.address_2 = user[:address2]
+    u.postcode = user[:postcode]
+    u.city_id = city.id
```
Comment (gtm19) [inline db/seeds.rb:105]: I think you can do 
```ruby
User.new(user)
```
since `user` is a hash. Will need to add the password separately still.

**Review by gtm19 — CHANGES_REQUESTED** [review]

File: db/seeds.rb (hunk 1 above)
Comment (akilchomoko) [reply db/seeds.rb:105]: Okay, makes sense. Will do when local as part of refactoring.

Comment (akilchomoko) [reply db/seeds.rb:105]: Will updated locally. I can't do a push at the moment, as I need to pull due to merge conflict. Can you approve the PR, so I can pull down and resolve it.

**Review by gtm19 — APPROVED** [review]

**Pull request merged**